src/Microsoft.DocAsCode.Build.Common/Reference/ApplyOverwriteDocument.cs

```diff
@@ -140,7 +140,9 @@ select new
                 }
                 catch (YamlException ye)
                 {
-                    throw new DocumentException($"Unable to deserialize YAML header from \"{s.Documentation.Path}\" Line {s.Documentation.StartLine} to TYPE {typeof(T).Name}: {ye.Message}", ye);
+                    var message = $"Unable to deserialize YAML header from \"{s.Documentation.Path}\" Line {s.Documentation.StartLine} to TYPE {typeof(T).Name}: {ye.Message}";
+                    Logger.LogError(message, code: ErrorCodes.Build.InvalidYamlHeader);
```

Comment:
This is a different scenarion [overwrite file](https://dotnet.github.io/docfx/tutorial/intro_overwrite_files.html). Change to `invalidOverwriteFile`.